examples/ido-pool/programs/ido-pool/src/lib.rs

```diff
@@ -0,0 +1,356 @@
+use anchor_lang::prelude::*;
+use anchor_lang::solana_program::program_option::COption;
+use anchor_spl::token::{self, Burn, Mint, MintTo, TokenAccount, Transfer};
+
+// Assume that all tokens use 6 decimal places
```

Comment:
Is this still an assumption?

Reply:
Not anymore, I'd added these lines to ensure the decimals are the same. Those two lines with the comments can be deleted. I'm trying to push an update that deletes those two lines but I'm getting a merge conflict. Forgot to pull before making changes, ugh.
```
    pub redeemable_mint: CpiAccount<'info, Mint>,
    #[account("usdc_mint.decimals == redeemable_mint.decimals")]
    pub usdc_mint: CpiAccount<'info, Mint>,
```

Reply:
Actually, just seeing now. I think you already removed the comments. So that is all good.